### PR TITLE
Update henry.py

### PR DESCRIPTION
**Pull request by wanda-carlson**
with body

### CODE REVIEW

File: src/henry.py [hunk 1]
```diff
@@ -2,4 +2,6 @@
 print("Henry testing v0.8.0")
 print("Henry testing v0.8.0")
 print("Wandaaaa testing v0.8.0")
+print("Henry testing v0.8.0")
```
Comment (wanda-carlson) [inline src/henry.py:5]: commmit
```suggestion
print("Henry testing v0.8.01")
```

File: src/henry.py [hunk 2]
```diff
@@ -2,4 +2,6 @@
 print("Henry testing v0.8.0")
 print("Henry testing v0.8.0")
 print("Wandaaaa testing v0.8.0")
+print("Henry testing v0.8.01")
```
Comment (henry-richard) [inline src/henry.py:5]:
```suggestion
print("Henry testing v0.8.21")
```

File: src/henry.py [hunk 3]
```diff
@@ -2,4 +2,6 @@
 print("Henry testing v0.8.0")
 print("Henry testing v0.8.0")
 print("Wandaaaa testing v0.8.0")
+print("Henry testing v0.8.21")
+print("Wandaaaa testing v0.8.0")
```
Comment (henry-richard) [inline src/henry.py:6]:
```suggestion
print("Nooooooo testing v0.8.0")
```

**Review by henry-richard — CHANGES_REQUESTED** [review]

**Comment by wanda-carlson** [comment]
comment from github

**Comment by wanda-carlson** [comment]
comment from slack

**Comment by wanda-carlson** [comment]
Testing

**Comment by wanda-carlson** [comment]
Testing 2

**Pull request merged**